municipalities/E/1/E151.json

```diff
@@ -1,8 +1,8 @@
 {
   "codiceProvincia": "",
   "codiceRegione": "",
-  "denominazione": "Gravedona ed Uniti",
-  "denominazioneInItaliano": "Gravedona ed Uniti",
+  "denominazione": "Gravedona",
```

Comment:
I think you should upgrade even the origin csv in case these files are generated again from the source

Reply:
these files are auto-generated using this https://github.com/pagopa/io-services-metadata/blob/master/src/config.ts#L6

Reply:
This is where we retrieve all municipalities data

https://github.com/pagopa/io-services-metadata/blob/cc06954d7d35563346f6e6bb7e8f7fc4ccce3ab3/src/config.ts#L5

So next time we launch this script it will be overwritten

Reply:
So you say that we need to change the script to manage our special case?

I don't think that I can change the original file :) https://www.istat.it/storage/codici-unita-amministrative/Elenco-comuni-italiani.csv

Reply:
then I guess there's a risk that this will be overwritten

Reply:
ok I changed this csv municipalities_with_catastale.csv, this is correct?

https://github.com/pagopa/io-services-metadata/blob/master/data/municipalities/municipalities_with_catastale.csv

Reply:
I follow this instructions
https://github.com/pagopa/io-services-metadata#updating-municipalities-metadata

also I updated this file https://github.com/pagopa/io-services-metadata/blob/master/data/municipalities/municipalities_with_catastale.csv

Reply:
Yes. it works because `E151` municipality doesn't exist anymore on this archive
https://www.istat.it/storage/codici-unita-amministrative/Elenco-comuni-italiani.csv
So it won't be overwritten